scr/Parking-Zone.MVC/Areas/Admin/Views/ParkingSlots/Create.cshtml

```diff
@@ -0,0 +1,77 @@
+﻿
+@model SlotCreateVM
+
+@{
+    ViewData["Title"] = "Create Slot";
+}
+<h1>@Model.ParkingZoneName</h1>
+
+<h2>Create Slot</h2>
+
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Create" asp-controller="ParkingSlots" method="post">
+            <div class="form-group">
+                <label asp-for="Number"></label>
+                <input asp-for="Number" class="form-control" />
+                <span asp-validation-for="Number" class="text-danger"></span>
+            </div>
+
+            <div class="form-group">
+                <label asp-for="IsAvailable"></label>
+                <select asp-for="IsAvailable" class="form-control">
+                    <option value="false">False</option>
+                    <option value="true">True</option>
+                </select>
+                <span asp-validation-for="IsAvailable" class="text-danger"></span>
+            </div>
+
+            <div class="form-group">
+                <label asp-for="Category"></label>
+                <select asp-for="Category" class="form-control" id="categorySelect">
+                    <option value="">-- Select Category --</option>
```

Comment:
`<select asp-for="Category" class="form-select" asp-items="Html.GetEnumSelectList<SlotCategoryEnum>()"></select> bir qatorgina`